docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -50,7 +50,7 @@ The following list provides the Kaldi\*-specific parameters.
 
 ```sh
 Kaldi-specific parameters:
-  --counts COUNTS       A file name with full path to the counts file
+  --counts COUNTS       A file name with full path to the counts file or empty string id counts are included in model file
```

Comment:
I have couple questions:
1. why will it be bad if we always apply count values that are specified inside the model. So it is not needed to invoke `--counts` option explicitly with same empty string.
2. what will be if both count values specified through both `--counts` and inside the model. Which count values will be applied.

Reply:
1. Tool from Kaldi that we use for reference results have separate option for applying counts. By default it is turned off in this tool.  Also we have models that have counts values inside model file, but we converted it without counts. I don't want change  behavior for old models.
2. Values from file will be applied.